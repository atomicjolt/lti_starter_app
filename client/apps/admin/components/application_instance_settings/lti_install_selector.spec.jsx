import React from 'react';
import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';

import LtiInstallSelector from './lti_install_selector';

const mockStore = configureStore([]);
const store = mockStore({
  ltiInstallKeys: {},
});

describe('LtiInstallSelector', () => {
  it('renders the LtiInstallSelector component', () => {
    const result = TestRenderer.create(
      <Provider store={store}>
        <LtiInstallSelector/>
      </Provider>
    );
    expect(result).toMatchSnapshot();
  });
});
