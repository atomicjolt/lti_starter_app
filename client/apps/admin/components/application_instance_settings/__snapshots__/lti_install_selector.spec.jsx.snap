// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LtiInstallSelector renders the LtiInstallSelector component 1`] = `
<div
  className="c-input--container"
>
  <table
    className="c-table c-table--instances"
  >
    <thead>
      <tr>
        <th
          className="sortable-header"
        >
          <div
            className="aj-flex"
          >
            <button
              className="aj-sort-btn"
              onClick={[Function]}
              type="button"
            >
              <span
                className="aj-flex"
              >
                ID
                <i
                  className="material-icons aj-icon-btn-sort"
                >
                  
                </i>
              </span>
            </button>
          </div>
        </th>
        <th
          className="sortable-header"
        >
          <div
            className="aj-flex"
          >
            <button
              className="aj-sort-btn"
              onClick={[Function]}
              type="button"
            >
              <span
                className="aj-flex"
              >
                ISS
                <i
                  className="material-icons aj-icon-btn-sort"
                >
                  
                </i>
              </span>
            </button>
          </div>
        </th>
        <th
          className="sortable-header"
        >
          <div
            className="aj-flex"
          >
            <button
              className="aj-sort-btn"
              onClick={[Function]}
              type="button"
            >
              <span
                className="aj-flex"
              >
                CLIENT ID
                <i
                  className="material-icons aj-icon-btn-sort"
                >
                  
                </i>
              </span>
            </button>
          </div>
        </th>
        <th
          className="sortable-header"
        >
          <div
            className="aj-flex"
          >
            <button
              className="aj-sort-btn"
              onClick={[Function]}
              type="button"
            >
              <span
                className="aj-flex selected"
              >
                CREATED
                <i
                  className="material-icons aj-icon-btn-sort"
                >
                  arrow_downward
                </i>
              </span>
            </button>
          </div>
        </th>
        <th />
      </tr>
    </thead>
    <tbody />
  </table>
  <ul
    className="pagination"
  >
    <li
      className="previous disabled"
    >
      <a
        aria-disabled="true"
        aria-label="Previous page"
        className=" "
        onClick={[Function]}
        onKeyPress={[Function]}
        rel="prev"
        role="button"
        tabIndex="-1"
      >
        Prev
      </a>
    </li>
    <li
      className="next"
    >
      <a
        aria-disabled="false"
        aria-label="Next page"
        className=""
        onClick={[Function]}
        onKeyPress={[Function]}
        rel="next"
        role="button"
        tabIndex="0"
      >
        Next
      </a>
    </li>
  </ul>
</div>
`;

exports[`lti_install_selector renders the lti_install_selector component 1`] = `<lti_install_selector />`;
